packages/foreman/tfm/tfm.spec

```
@@ -61,6 +61,7 @@ Obsoletes: ruby193-ruby-wrapper
 
 # Obsolete packages no longer carried within this SCL to both clean up
 # and to ensure smooth upgrades when old packages deps aren't satisified
+Obsoletes: %{scl_prefix_ror}rubygem-jquery-turbolinks < 3.0
```

IMHO it's not very clean to obsolete something from another collection here. However, this particular example is also from the regular scl. It's also custom to pin on `Version-(Release+1)`. See https://github.com/theforeman/foreman-packaging/tree/rpm/develop#howto-removing-a-package
```suggestion
Obsoletes: %{scl_prefix}rubygem-jquery-turbolinks < 2.1.0-9
```